tests/testthat/fixtures/test-rebuild.R

```diff
@@ -0,0 +1,37 @@
+#' Rebuild fixtures
+#'
+#' @examples
+#' testthat::test_dir(testthat::test_path("fixtures"))
+#'
```

Comment:
I added a "test" file in the `tests/testthat/fixtures` directory to rebuild the relevant packfiles. Happy to retool this if you have a different style in mind, I just wanted to make sure the process was reproducible.